auto: True
drone_ids : [1,2]
leader_start_point: [3, 3, 3]
follower_start_point: [3, 3, 3]
take_off_height: 3
take_off_blocking: False
inspect: [0,0,0,6.5] # x,y,z,r

relative_angle: 0.7

waypoints: 
  - [0, 5, 5]
  - [5, 0, 5]
  - [0, -5, 5]
  - [-5, 0, 5]
  - [0, 5, 5]
  - [5, 0, 5]
  - [0, -5, 5]
  - [-5, 0, 5]